README.md

```diff
@@ -1,4 +1,4 @@
-# ![teams logo](https://github.com/microsoft/teams-ai/assets/14900841/10a0fbac-9d31-46d5-ad65-0eeb33b787a8) Teams AI Library
+# ![teams logo](https://media.githubusercontent.com/media/microsoft/teams-ai/main/assets/icon.png) Teams AI Library
```

Comment:
In the preview, this shows up as extremely large - does it need to be resized?

Reply:
good catch, updated to have a fixed width